lib/net/Makefile

```diff
@@ -5,20 +5,33 @@ NETIFFILES := $(filter-out $(LWIPDIR)/netif/slipif.c,$(NETIFFILES))
 LWIPSRCS := $(COREFILES) \
             $(CORE4FILES) \
             $(CORE6FILES) \
-            $(NETIFFILES) \
-            $(APIFILES)
+            $(APIFILES) \
+            $(NETIFFILES)
 
 # Include driver sources
 DRIVERSRCS := $(NXDK_DIR)/lib/net/pktdrv/pktdrv.c \
               $(NXDK_DIR)/lib/net/nforceif/src/driver.c \
               $(NXDK_DIR)/lib/net/nforceif/src/sys_arch.c
 
-SRCS += $(LWIPSRCS) $(DRIVERSRCS)
+NET_SRCS := \
+	$(LWIPSRCS) \
+	$(DRIVERSRCS)
+
+NET_OBJS = $(addsuffix .obj, $(basename $(NET_SRCS)))
 
-# Add includes
 CFLAGS += -I $(LWIPDIR)/include
 CFLAGS += -I $(NXDK_DIR)/lib/net/nforceif/include
 CFLAGS += -I $(NXDK_DIR)/lib/net/pktdrv
 CXXFLAGS += -I $(LWIPDIR)/include
 CXXFLAGS += -I $(NXDK_DIR)/lib/net/nforceif/include
 CXXFLAGS += -I $(NXDK_DIR)/lib/net/pktdrv
```

Comment:
I believe this must be `NXDK_` prefixed; we do this, so the user can overwrite `CFLAGS` and friends with their own stuff, without breaking compilation.
Also relevant for people who only run the makefile to extract the NXDK variables, so they can hook their own build-system into it.

Reply:
These days, we also do these without space between `-I` and the path